onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -14,9 +14,12 @@ static const std::string kMaxPartitionIterations = "ORT_TENSORRT_MAX_PARTITION_I
 static const std::string kMinSubgraphSize = "ORT_TENSORRT_MIN_SUBGRAPH_SIZE";
 static const std::string kMaxWorkspaceSize = "ORT_TENSORRT_MAX_WORKSPACE_SIZE";
 static const std::string kFP16Enable = "ORT_TENSORRT_FP16_ENABLE";
+static const std::string kINT8Enable = "ORT_TENSORRT_INT8_ENABLE";
+static const std::string kINT8CalibrationTableName = "ORT_TENSORRT_INT8_CALIBRATION_TABLE_NAME";
+static const std::string kINT8UseTensorrtCalibrationTable = "ORT_TENSORRT_INT8_USE_TENSORRT_CALIBRATION_TABLE";
 static const std::string kDumpSubgraphs = "ORT_TENSORRT_DUMP_SUBGRAPHS";
 static const std::string kEngineCacheEnable = "ORT_TENSORRT_ENGINE_CACHE_ENABLE";
-static const std::string kEngineCachePath = "ORT_TENSORRT_ENGINE_CACHE_PATH";
+static const std::string kCachePath = "ORT_TENSORRT_CACHE_PATH";
```

Comment:
this is a compatibility breaking change.
users who are already using previous name, may not know it has changed.
i think it would be safer to add another check to throw error if the previous name is found set in the environment variables.
or we don't change the name at all.

Reply:
you can handle this in a subsequent PR?